charts/argo-workflows/values.yaml

```diff
@@ -536,9 +536,9 @@ server:
     # - name: FOO
     #   value: "bar"
 
-  # -- Auth Mode is available from `server` , `client` or `sso`. If you chose `sso` , please configure `.Values.server.sso` as well.
+  # -- A list of supported authentication modes. Available values are `server`, `client`, or `sso`. If you provide sso, please configure `.Values.server.sso` as well.
   ## Ref: https://argoproj.github.io/argo-workflows/argo-server-auth-mode/
-  authMode: ""
+  authMode: []
```

Comment:
Great change.  Thank you.

Could this break existing users who have it set to a string?  Should we add something in `NOTES.txt` about a deprecation and handle both for a bit?  I don't know if it is worth it. wdyt? 🤔